src/components/dashboard/FaceVerification/components/Instructions.js

```diff
@@ -31,7 +38,13 @@ const Dot = () => (
   </Text>
 )
 
-const Instructions = ({ styles, onDismiss = noop }) => (
+const CheckMark = () => (
+  <Text color="white" fontSize={normalize(20)}>
+    &#10003;{' '}
```

Comment:
we have icons

Reply:
you mean this package: react-native-vector-icons/lib/create-icon-set-from-fontello ?

Reply:
the icon I found called check does not look similar to the design (components/common/view/icon)